executor/modes/local/runner.py

```
@@ -0,0 +1,407 @@
+                    )
+                except asyncio.TimeoutError:
+                    continue
+
+                # Process the task
+                try:
+                    self.current_task = task_data
+                    await self._execute_task(task_data)
+                except Exception as e:
+                    task_id = task_data.get("task_id", -1)
+                    logger.exception(
+                        f"Task execution failed for task_id={task_id}: {e}"
```

Ruff (0.14.14)

Redundant exception object included in `logging.exception` call

(TRY401)